modules/contracts/adapters/dex/src/exchanges/exchange_resolver.rs

```diff
@@ -33,19 +33,19 @@ pub(crate) fn identify_exchange(value: &str) -> Result<Box<dyn Identify>, DexErr
 
 pub(crate) fn resolve_exchange(value: &str) -> Result<&'static dyn DexCommand, DexError> {
     match value {
-        #[cfg(feature = "juno")]
+        #[cfg(feature = "wynd")]
         crate::exchanges::junoswap::JUNOSWAP => Ok(&crate::exchanges::junoswap::JunoSwap {}),
-        #[cfg(feature = "juno")]
+        #[cfg(feature = "wynd")]
         abstract_wyndex_adapter::WYNDEX => Ok(&abstract_wyndex_adapter::dex::WynDex {}),
         #[cfg(feature = "osmosis")]
         abstract_osmosis_adapter::OSMOSIS => Ok(&abstract_osmosis_adapter::dex::Osmosis {
             local_proxy_addr: None,
         }),
-        #[cfg(feature = "terra")]
```

Comment:
Can we make this terraswap instead of terra?

Reply:
Sure